packages/core/src/index.ts

```diff
@@ -26,7 +31,8 @@ export {
   sendTo,
   sendParent,
   forwardTo,
-  interpret,
+  createActor,
+  interpret, // deprecated
   Interpreter,
   ActorStatus as InterpreterStatus,
```

Comment:
`InterpreterStatus` should be phased out (or it should just be an alias of `ActorStatus` with the `@deprecated` tag). Other packages should start depending on the `ActorStatus` itself.